builder/internals.go

```diff
@@ -489,8 +489,11 @@ func (b *Builder) processImageFrom(img *imagepkg.Image) error {
 		b.Config = img.Config
 	}
 
-	if len(b.Config.Env) == 0 {
-		b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
+	// No default path on Windows (set by HCS)
+	if runtime.GOOS != "windows" {
+		if len(b.Config.Env) == 0 {
+			b.Config.Env = append(b.Config.Env, "PATH="+daemon.DefaultPathEnv)
```

Comment:
This is probably correct, but I'd like to understand... why is there no default PATH on Windows?